sdk/docs/iot/README.md

```diff
@@ -236,6 +245,19 @@ void my_telemetry_func(void)
 
 This project welcomes contributions and suggestions. Find more contributing details [here](https://github.com/Azure/azure-sdk-for-c/blob/main/CONTRIBUTING.md).
 
+If you wish to contribute with a sample please read the guideline below:
+
+1. Sample owners are fully responsible for their sample code, and any dependency they might have,
```

Comment:
```suggestion
1. Sample owners are fully responsible for their sample code, and any dependency they might have
```
nit: have all line endings in the bullet list have the same format (either `.` or nothing)